bedrock/base/templates/includes/news-feed.html

```diff
@@ -6,7 +6,7 @@
 {% if LANG.startswith('en') and articles %}
 <section class="news">
   <div class="content">
-    <h2 class="section-title"><span>{{ _('Fresh from our Blog') }}</span></h2>
+    <h2 class="section-title"><span>{{ _('Fresh From Our Blog') }}</span></h2>
```

Comment:
Given the l10n implications and work already progressing elsewhere, I'll undo this change.